database/mysql-ob-berlin/requirements.yaml

```diff
@@ -11,9 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: {{ .Values.kubernetes.serviceAccount }}
-  namespace : {{ .Release.Namespace }}
+dependencies:
+  - name: mysql
+    version: "1.6.9"
+    repository: "https://helm.wso2.com"
```

Comment:
@nalakagunathilaka @Aaquiff have we uploaded the base MySQL Helm chart to `https://helm.wso2.com` for this to work?

Reply:
Yes.

Reply:
@chirangaalwis it has been uploaded.